src/js/events/SystemLogActions.js

```diff
@@ -0,0 +1,118 @@
+import {
+  REQUEST_SYSTEM_LOG_SUCCESS,
+  REQUEST_SYSTEM_LOG_ERROR,
+  REQUEST_PREVIOUS_SYSTEM_LOG_SUCCESS,
+  REQUEST_PREVIOUS_SYSTEM_LOG_ERROR
+} from '../constants/ActionTypes';
+import AppDispatcher from './AppDispatcher';
+import CookieUtils from '../utils/CookieUtils';
+import SystemLogUtil from '../utils/SystemLogUtil';
+
+/**
+ * Implementation of server sent event handling for
+ * https://github.com/dcos/dcos-log
+ */
+
+// Store of current open connections
+let sources = {};
+const SystemLogActions = {
+  subscribe(nodeID, options = {}) {
+    let {
+      subscriptionID
+    } = options;
+
+    let url = SystemLogUtil.getUrl(nodeID, options);
+    subscriptionID = subscriptionID || global.btoa(url);
+    let source = new EventSource(url, {
+      withCredentials: Boolean(CookieUtils.getUserMetadata())
```

Comment:
Let's not use the Boolean object to cast non-falsy values to booleans and rather add a an additional condition as it's easier to reason about.